core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -484,9 +518,9 @@ object StorageTool extends Logging {
 
 
   def buildMetadataProperties(
-    clusterIdStr: String,
-    config: KafkaConfig
-  ): MetaProperties = {
+                               clusterIdStr: String,
+                               config: KafkaConfig
+                             ): MetaProperties = {
```

Comment:
Please preserve the old indentation. In Kafka that is how we indent parameters to a method that span multiple lines.